shared/src/Datadog.Trace.ClrProfiler.Native/crashhandler.cpp

```diff
@@ -0,0 +1,409 @@
+#include "crashhandler.h"
```

Comment:
shouldn't we add the header ?

Reply:
Apparently we have no headers anywhere in the native loader, interesting